packages/templates/src/home/Explorer.tsx

```diff
@@ -504,53 +505,109 @@ export const Template: VFC<
-                  <InputGroup>
-                    <NumberInput
-                      clampValueOnBlur={false}
-                      min={0}
-                      step={Math.pow(10, -currency.decimals)}
-                      precision={currency.decimals}
-                      allowMouseWheel
-                      w="full"
-                      isDisabled={isSubmitting}
-                      onChange={(x: any) => setValue('minPrice', x)}
-                      format={(e) => e.toString()}
-                    >
-                      <NumberInputField
-                        placeholder={t('explore.form.min-price.placeholder')}
-                        {...register('minPrice')}
-                      />
-                      <NumberInputStepper>
-                        <NumberIncrementStepper />
-                        <NumberDecrementStepper />
-                      </NumberInputStepper>
-                    </NumberInput>
-                  </InputGroup>
-
-                  <InputGroup>
-                    <NumberInput
-                      clampValueOnBlur={false}
-                      min={0}
-                      step={Math.pow(10, -currency.decimals)}
-                      precision={currency.decimals}
-                      allowMouseWheel
-                      w="full"
-                      isDisabled={isSubmitting}
-                      onChange={(x: any) => setValue('maxPrice', x)}
-                      format={(e) => e.toString()}
-                    >
-                      <NumberInputField
-                        placeholder={t('explore.form.max-price.placeholder')}
-                        {...register('maxPrice')}
-                      />
-                      <NumberInputStepper>
-                        <NumberIncrementStepper />
-                        <NumberDecrementStepper />
-                      </NumberInputStepper>
-                    </NumberInput>
-                  </InputGroup>
-                </Flex>
+                <FormControl isInvalid={!!errors.minPrice || !!errors.maxPrice}>
+                  <Flex gap={3}>
+                    <InputGroup>
+                      <NumberInput
+                        clampValueOnBlur={false}
+                        step={Math.pow(10, -currency.decimals)}
+                        allowMouseWheel
+                        w="full"
+                        isDisabled={isSubmitting}
+                        onChange={(x: any) => setValue('minPrice', x)}
+                        format={(e) => e.toString()}
+                      >
+                        <NumberInputField
+                          id="minPrice"
+                          placeholder={t('explore.form.min-price.placeholder')}
+                          {...register('minPrice', {
+                            validate: (value) => {
+                              if (!value) return
+                              const splitValue = value.toString().split('.')
```

Comment:
Why only the validations in this file need to convert the value to a string?

Reply:
Seems if these are not numbers it creates issues with the filters logic. I tried to change it to strings but seems to create more problems, might be better to create a different issue if needed